SCF-Session-8-Assignment/src/question/Reptile.java

```diff
@@ -0,0 +1,18 @@
+package question;
+
+/**
+ * This class is for category Reptile
+ * @author Jugal
+ * Dated 07/23/2019
+ */
+public abstract class Reptile extends Animal {
+

```

Comment:
Declare some different properties of Reptile class and initialize them in Reptile class constructor and initialize Animal class property using super().

Reply:
Done